### DISCUSSION TITLE
Show sql-requests with create_engine(..., echo=True)

### DISCUSSION THREAD

**Discussion by GerasimovRM** [collerek/ormar#345] [Q&A] [answered]
Hi there! I began to use fastapi with ormar. When I create engine with argement echo=True, application don't show sql-requests. How i can do it? I have tried run application with other mods (release/debug), and added logger. All attempts is failed. 
My code (SQL_ECHO=True):

```python
import ormar
from config import DATABASE_URL, SQL_ECHO
import databases
import sqlalchemy

metadata = sqlalchemy.MetaData()
database = databases.Database(str(DATABASE_URL))
engine = sqlalchemy.create_engine(str(DATABASE_URL), echo=SQL_ECHO)
```

Thanks in advance!

**Comment by collerek** [comment, marked as answer]
That's because the `sqlalchemy` is not used to run the queries.

`ormar` runs the queries with `encode\databases` so you need to search for a debug feature there, not sure if it's implemented, you need to check the source code as the documentation is super basic.

If there is no debug mode in databases it can probably be an enhancement proposal for `ormar`.

**Reply by lewoudar** [reply]
Hi, I hope this feature will come one day :)

**Reply by rawberg** [reply]
```
import logging
logging.basicConfig()
logging.getLogger('databases').setLevel(logging.DEBUG)
```
https://github.com/encode/databases/issues/397

**Reply by lewoudar** [reply]
Thanks @rawberg !
